### PR TITLE
Scripts for IBM Pipelines

### PR DESCRIPTION
**Pull request by feloy**
**What type of PR is this?**

/kind tests

**What does this PR do / why we need it**:

This PR adds scripts to be executed from IBM Pipelines to run unit tests and integration tests on IBM Cloud

### CODE REVIEW

File: .ibm/pipelines/kubernetes-tests.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    set -e
+    make install
+    make test-integration-devfile
+    make test-e2e-devfile
+    make test-cmd-project
+) |& tee "/tmp/${LOGFILE}"
+RESULT=${PIPESTATUS[0]}
+
+install_gh
+save_logs "${LOGFILE}" "Kubernetes Tests"
```
Comment (anandrkskd) [inline .ibm/pipelines/kubernetes-tests.sh:23]: @feloy this save log is added as a comment on gihub by itself or do we need to enable anything on pipleline side

**Review by anandrkskd — APPROVED** [review]
/lgtm

File: .ibm/pipelines/kubernetes-tests.sh (hunk 1 above)
Comment (feloy) [reply .ibm/pipelines/kubernetes-tests.sh:23]: All is done here, there is nothing else to be done (except getting a private key to connect to GitHub)

**Review by rnapoles-rh — COMMENTED** [review]
/lgtm

**Pull request merged**